### PR TITLE
Add copy button to link under Access event information via REST API

### LINKED ISSUE #2069: Add copy button to link under  "Access event information via REST API"
**Describe the bug**

Currently the user has to manually copy the link under Add copy button to link under  "Access event information via REST API" .
**To Reproduce**
1. Go to 'Export'
3. Scroll down to 'You can make a GET request to the following endpoint to access the selected information'

**Expected behavior**

Add copy button to link under  "Access event information via REST API" 
**Screenshots**

screenshot 2019-01-31 at 11 46 55 pm

**Additional context**

I'm working on it.

### PR DESCRIPTION
**Pull request by harsh-jindal**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Add copy button to link under Access event information via REST API

#### Changes proposed in this pull request:

screenshot 2019-02-04 at 10 54 29 pm

Fixes #2069

### CODE REVIEW

File: app/templates/components/events/view/export/api-response.hbs [hunk 1]
```diff
@@ -31,9 +31,18 @@
...
+  clipboardText=displayUrl
+  class='ui secondary segment url copy-url'
+  click=(action 'copiedUrl')
+}}
+  <div class="display-url">
+    <pre>{{displayUrl}}</pre>
+  </div>
+{{/copy-button}}
+{{#if this.isLinkSuccess}}
+  <div><span class="display-success">Link copied to clipboard</span></div>
```
Comment (niranjan94) [inline app/templates/components/events/view/export/api-response.hbs]: Could you just use a tooltip for this ?

**Comment by harsh-jindal** [comment]
@niranjan94 Made the changes.

File: app/styles/components/events/view/export.scss [hunk 2]
```diff
@@ -32,3 +32,11 @@
 .url {
   padding: 0 1em !important;
 }
+
+.copy-url {
```
Comment (niranjan94) [inline app/styles/components/events/view/export.scss]: use the `fluid` class instead of this

Comment (harsh-jindal) [reply app/styles/components/events/view/export.scss]: @niranjan94 If I use the class fluid along with the button class, On hover the colors are inverted.
{{#copy-button
    clipboardText=displayUrl
    class='fluid ui secondary segment url button'
  }}
And if I dont use the button class fluid class doesn't work.
This is how it looks on hover : 
screenshot 2019-02-05 at 9 02 10 pm

I haven't made these changes yet. Shall I change it to this?

File: app/styles/components/events/view/export.scss [hunk 3]
```diff
@@ -32,3 +32,11 @@
...
+  width: 100%;
+}
+
+.display-url {
```
Comment (CosmicCoder96) [inline app/styles/components/events/view/export.scss]: Remove this class. On mobile view it causes this:
image

File: app/styles/components/events/view/export.scss (hunk 2 above)
Comment (CosmicCoder96) [inline app/styles/components/events/view/export.scss]: just remove this class @harsh-jindal

**Comment by CosmicCoder96** [comment]
@harsh-jindal can you update this ?

**Comment by harsh-jindal** [comment]
@CosmicCoder96 Will do it an hour.

File: app/styles/components/events/view/export.scss (hunk 2 above)
Comment (harsh-jindal) [reply app/styles/components/events/view/export.scss]: @CosmicCoder96 Text will still overflow If i remove that class. Instead I have added media breakpoints to handle overflow.

Comment (CosmicCoder96) [reply app/styles/components/events/view/export.scss]: @harsh-jindal is the text-overflow being caused by the changes you made or is it a pre-existing issue?

Comment (harsh-jindal) [reply app/styles/components/events/view/export.scss]: @CosmicCoder96 This issue was not there was before as class="ui segment secondary" took care of it. But #copy-button causes the overflow issue. That is why I have to explicitly define a class.

**Pull request merged**